datafusion/common/src/config.rs

```diff
@@ -223,6 +223,11 @@ config_namespace! {
 
         /// Parquet options
         pub parquet: ParquetOptions, default = Default::default()
+
+        /// Fan-out during physical planning.
+        ///
+        /// This is mostly use to plan `UNION` children in parallel.
+        pub physical_planning_fanout: usize, default = 10
```

Comment:
I recommend calling this "planning_concurrency"

Reply:
done